code/game/data_huds.dm

```diff
@@ -244,8 +244,9 @@
 	holder.icon_state = "hudno_id"
 	holder.icon =  'icons/mob/hud.dmi'  					// FULP: Reset to default icon, in case you swapped to a FULP .dmi but we need to restore it.  **THIS IS TEMPORARY**
 	if(wear_id?.GetID())
-		holder.icon = wear_id.GetID().return_icon_hud()   	// FULP ADDITION - This asks the ID to ask its associated Job what icon to use.
-		holder.icon_state = "hud[ckey(wear_id.GetJobName())]"
+		var/obj/item/card/id/our_id = wear_id.GetID()
+		holder.icon = our_id.return_icon_hud()   	// FULP ADDITION - This asks the ID to ask its associated Job what icon to use.
+		holder.icon_state = "hud[ckey(wear_id.GetJobName())]" //See above
```

Comment:
You'll see this a lot too. Because we use nonstatic typing, doing thing.proc().proc() makes the linter flip its shit.